client/src/components/Description.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Description extends React.Component {
+  constructor(props) {
+    super(props);
+
+    let displayDescription = this.props.description;
+    if (displayDescription.length > 259) {
+      displayDescription = displayDescription.slice(0, 260) + '...';
+    };
+    console.log(displayDescription);
+    this.state = {
+      displayDescription: displayDescription,
+      showAll: false,
+    };
+    this.toggleDescription = this.toggleDescription.bind(this);
+  }
+
+  toggleDescription(event) {
+    event.preventDefault();
+    this.setState({
+      showAll: !this.state.showAll,
+    });
+  }
+
+  render() {
+    const isLengthy = this.state.displayDescription !== this.props.description;
```

Comment:
Should this be a const?

Reply:
Great catch! Didn't realize that.

Reply:
Once false or true I don't think it can dynamically change?